package.json

```diff
@@ -16,6 +16,26 @@
   "author": "brian@hovercraftstudios.com",
   "license": "MIT",
   "devDependencies": {
-    "lerna": "^2.1.2"
+    "@briancavalier/assert": "^3.2.0",
+    "babel-core": "^6.26.0",
```

Comment:
ive hoisted all common packages to the root `package.json` - according to lerna team this should result in faster bootstraps

Reply:
Cool.  I was under the impression that each package had to list its own deps and then use `bootstrap --hoist`.  This is much better.

Reply:
I believe that each pkg should still lost its own deps, at least i treat lerna packages being in single repository just some kimd of implementation detail. Ideally i always want to keep them separate so the pkg's dir could copied/moved and still keep working without changes.

Also each pkg need to have its deps with executables listed so they can be linked properly and used from npm scripts

Gathering all deps in root package.json is an additional step which according to lerna docs (i think) allows lerna for faster bootstraps but i aint sure why.

Reply:
https://github.com/lerna/lerna/blob/master/FAQ.md#root-packagejson

> The root can also hold all the "hoisted" packages, which speeds up bootstrapping when using the --hoist flag.